bokehjs/src/lib/models/layouts/grid_box.ts

```diff
@@ -9,6 +9,8 @@ export class GridBoxView extends LayoutDOMView {
   connect_signals(): void {
     super.connect_signals()
     this.connect(this.model.properties.children.change, () => this.rebuild())
+    this.connect(this.model.properties.rows.change, () => this.rebuild())
+    this.connect(this.model.properties.cols.change, () => this.rebuild())
```

Comment:
This could be shortened and made more meaningful with (indicate that it's the same action in all cases):
```ts
const {children, rows, cols} = this.model.properties
this.on_change([children, rows, cold], () => this.rebuild())
```

Reply:
Add `spacing` as well?

Reply:
Yes.

Reply:
I'll add spacing and the regression test

Reply:
done